docs/partials/api.md

```diff
@@ -5,10 +5,10 @@
 
 ### Default
 
-  <div class="exampleWrapper">
-    <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/default.html) -->
-    <!-- AURO-GENERATED-CONTENT:END -->
-  </div>
+<div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/default.html) -->
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
```

Comment:
Isn't the standard for this file to be maned `apiExamples.md`? 

I see that @jordanjones243 changed this to `api.md` in this commit. Shouldn't we be updating to meet the Generator standard? 

- https://github.com/AlaskaAirlines/auro-carousel/commit/16d883dd389146b28d228c416a3f7e73c998c92c

Reply:
@blackfalcon The Generator was recently updated to reflect the `index.md` and `api.md` demo file naming standard

Reply:
This example references has been causing a lot of confusion. Please see this Discussion as we must manage a strategy for updating all the repos once more to fit this change. 

https://github.com/AlaskaAirlines/WC-Generator/discussions/522